packages/dds/tree/src/feature-libraries/object-forest/objectForest.ts

```diff
@@ -74,9 +87,6 @@ export class ObjectForest extends SimpleDependee implements IEditableForest {
     }
 
     applyDelta(delta: Delta.Root): void {
```

Comment:
I think this warrants a doc comment as well.

Reply:
This is part of the EditableForest API and thus inherits documentation from the interface.